src/test/shared/telemetry/defaultTelemetryService.test.ts

```diff
@@ -86,24 +92,23 @@ describe('DefaultTelemetryService', function () {
         assert.strictEqual(mockPublisher.feedback, feedback)
     })
 
-    it('assertPassiveTelemetry() throws if active, non-cached metric is emitted during startup', async function () {
-        // Simulate cached telemetry by prepopulating records before start().
-        // (Normally readEventsFromCache() does this.)
-        service.record(fakeMetric({ value: 1, passive: true }))
-        service.record(fakeMetric({ value: 2, passive: true }))
-        // Active *cached* metric.
-        service.record(fakeMetric({ value: 4, passive: false }))
-        await service.start()
+    it('assertPassiveTelemetry() throws if it sees non-passive metrics in the general queue', async function () {
```

Comment:
The new description is technically correct. But why remove semantics from test descriptions? The intent is to check during startup. Yes, if we call assertPassiveTelemetry() elsewhere, then it would also throw. But I don't see why we would make test descriptions more vague. It reduces their value as documentation.

Reply:
See above. The function's tie to startup was the fact that it was checking the last value from the cache. With that removed, this becomes just a way to check the existing queue. The function needs to be renamed and possibly protected from being called more than once.